.travis.yml

```diff
@@ -12,7 +12,6 @@ notifications:
     on_success: always
     on_failure: always
     recipients:
-      - davetcoleman@gmail.com
```

Comment:
lol, i have this filtered to a folder i never look at so you don't have to bother. or you could just take me off permanently